Add support for multiple continuous input ports (#1218)
nest/nestml: doc/nestml_language/neurons_in_nestml.rst

Multiple input ports
^^^^^^^^^^^^^^^^^^^^

> If there is more than one line specifying a `spike` or `continuous` port with the same sign, a neuron with multiple receptor types is created. For example, say that we define three spiking input ports and two conitnuous currents as follows:

Reviewer: ```suggestion
If there is more than one line specifying a `spike` or `continuous` port with the same sign, a neuron with multiple receptor types is created. For example, say that we define three spiking input ports and two continuous currents as follows:
```

.. code-block:: nestml

   input:
       spikes1 <- spike
       spikes2 <- spike
       spikes3 <- spike
       I_stim1 <- continuous
       I_stim2 <- continuous

> For the sake of keeping the example simple, we assign a decaying exponential-kernel postsynaptic response to each spiking input port, each with a different time constant and the continuous currents added to say, membrane potential of soma (``V_m``) and distal (``V_md``) parts of the neuron:

Reviewer: ```suggestion
For the sake of keeping the example simple, we assign a decaying exponential-kernel postsynaptic response to each spiking input port, each with a different time constant. The continuous currents are added to, say, the membrane potential of the soma (``V_m``) and the distal parts of the neuron  (``V_d``):
```

       kernel I_kernel2 = exp(-t / tau_syn2)
       kernel I_kernel3 = -exp(-t / tau_syn3)
       inline I_syn pA = (convolve(I_kernel1, spikes1) - convolve(I_kernel2, spikes2) + convolve(I_kernel3, spikes3)) * pA
       V_m' = -(V_m - E_L) / tau_m + (I_syn + I_stim1) / C_m

>        V_d' = -(V_d - E_Ld) / tau_md + I_stim2 / C_m

Reviewer: ```suggestion
       V_d' = -(V_d - E_L) / tau_d + I_stim2 / C_m
```
